src/test/java/com/google/devtools/build/lib/packages/RuleClassTest.java

```diff
@@ -1035,13 +1039,74 @@ public void testRequiredToolchains() throws Exception {
             .add(attr("tags", STRING_LIST));
 
     ruleClassBuilder.addRequiredToolchains(
-        ImmutableList.of(
-            Label.parseAbsolute("//toolchain:tc1"), Label.parseAbsolute("//toolchain:tc2")));
+        Label.parseAbsolute("//toolchain:tc1"), Label.parseAbsolute("//toolchain:tc2"));
 
     RuleClass ruleClass = ruleClassBuilder.build();
 
     assertThat(ruleClass.getRequiredToolchains())
         .containsExactly(
             Label.parseAbsolute("//toolchain:tc1"), Label.parseAbsolute("//toolchain:tc2"));
   }
+
+  @Test
+  public void testExecutionPlatformConstraints_none() {
```

Comment:
Minor nit question: what's the current best practice on prefixing methods with "test" when we already have the @Test annotation? I know there's a movement to eliminate the redundancy but I'm not sure where that movement stands these days.